es.estoes.wallpaperDownloader.svg

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Ideally the icon, appdata and desktop launchers should be part of upstream repository.